typespec-tests/src/test/java/com/type/property/additionalproperties/ExtendsTests.java

```diff
@@ -104,4 +108,43 @@ public void testExtendsUnknownClient() {
         Assertions.assertEquals("ExtendsUnknownAdditionalProperties", properties.getName());
         Assertions.assertIterableEquals(propertyMap.entrySet(), properties.getAdditionalProperties().entrySet());
     }
+
+    @Test
+    @Disabled("Polymorphic deserialization doesn't support multiple levels of inheritance in Jackson, https://github.com/FasterXML/jackson-databind/issues/1188")
```

Comment:
I don't think this is the reason?

If you are not sure about it, just keep it empty.

Reply:
The reason for disable has been fixed in the new version.